core/src/kmx/kmx_file.cpp

```diff
@@ -68,11 +68,11 @@ KMX_BOOL KMX_ProcessEvent::LoadKeyboardFromBlob(
 
   *lpKeyboard = NULL;
 
-#ifdef KMX_64BIT
+#if defined(KMX_64BIT) || defined(__EMSCRIPTEN__)
   // allocate enough memory for expanded data structure + original data.
-  // Expanded data structure is double the size of data on disk (8-byte
-  // pointers) - on disk the "pointers" are relative to the beginning of
-  // the file.
+  // Expanded data structure is no more than double the size of data on
+  // disk (8-byte pointers) - on disk the "pointers" are relative to the
+  // beginning of the file.
```

Comment:
Isn't 8-byte pointers not true for emscripten because it's 32 bit? Or are these offsets and not pointers?